magento/magento2
# Bug fix for .lib-button-reset() not resetting text color

### Description (*)
Fixed issue where .lib-button-reset() resets the background but not the color.

Corrects issue where button text color does not provide enough contrast on standard background color (i.e. dark buttons with white text reset may present a white text on a white background).

### Manual testing scenarios (*)
1. Theme with the following variables set...
```
@primary__color: @color-black;
@page__background-color: @color-white;
@button__color: @color-white;
@button__background: @color-black;
```

2. On elements that use the `.lib-button-reset()` mixin such as the minicart "close" icon, it appears as a white "x" on a white background.


### Resolved issues:
1. [x] resolves magento/magento2#37900: Bug fix for .lib-button-reset() not resetting text color
